presto-native-execution/presto_cpp/main/http/tests/HttpTestBase.h

```diff
@@ -262,6 +270,8 @@ struct AsyncMsgRequestState {
   uint64_t maxWaitMillis{0};
   std::weak_ptr<StringPromise> msgPromise;
   RequestStatus requestStatus{kStatusUnknown};
+  std::function<void()> customFunc;
+  std::string name;
```

Comment:
it looks like you intended to remove this

Reply:
Oh yeah missed it. Thanks